.gitmodules

```diff
@@ -50,3 +50,6 @@
 [submodule "apps/resident-app"]
 	path = apps/resident-app
 	url = git@github.com:open-condo-software/condo-resident-app.git
+[submodule "apps/meters"]
+	path = apps/meters
```

Comment:
I guess the name is too generous...That's probably not the last app dedicated to meters domain. How about some clarification here?

Reply:
Any suggestions?
Maybe `meters-sync`?